integration/tests/git_source/git_source_submodule.rb

```diff
@@ -0,0 +1,69 @@
+require 'git_utils'
+require 'r10k_utils'
+require 'master_manipulator'
+test_name 'CODEMGMT-92 - C59238 - Single Git Source with Git Sub-module'

```

Comment:
Is this the correct test?  The ticket lists 59233 as a test that is included in this PR.